File: TypeScript/MongoDB/REST/src/rest-response.ts
class REST_Response {
    status_code: number;
    content_type: string;
    body: string;

    constructor(status_code: number, content_type: string, body: string) {
        this.status_code = status_code;
        this.content_type = content_type;
        this.body = body;
    }

    toString(): string {
        return `RestResponse(status_code=${this.status_code}, content_type='${this.content_type}', body='${this.body}')`;
    }

    to_dict(): { status_code: number; content_type: string; body: string } {
        return {
            status_code: this.status_code,
            content_type: this.content_type,
            body: this.body
        };
    }
}

export default REST_Response;

File: Java/Containers/MongoDB-REST/src/main/java/com/api/mongodb/rest/service/mongodb/rest/REST_Response.java
package com.api.mongodb.rest.service.mongodb.rest;

import java.util.HashMap;
import java.util.Map;

public class REST_Response {
    private final int statusCode;
    private final String contentType;
    private final String body;

    public REST_Response(int statusCode, String contentType, String body) {
        this.statusCode = statusCode;
        this.contentType = contentType;
        this.body = body;
    }

    public int getStatusCode() {
        return statusCode;
    }

    public String getContentType() {
        return contentType;
    }

    public String getBody() {
        return body;
    }

    @Override
    public String toString() {
        return "REST_Response(status_code=" + statusCode + ", content_type='" + contentType + "', body='" + body + "')";
    }

    public Map<String, Object> toDict() {
        Map<String, Object> map = new HashMap<>();
        map.put("status_code", statusCode);
        map.put("content_type", contentType);
        map.put("body", body);
        return map;
    }
}